apps/meteor/tests/e2e/page-objects/marketplace.ts

```diff
@@ -0,0 +1,45 @@
+import type { Locator, Page } from '@playwright/test';
+
+export class Marketplace {
+	private readonly page: Page;
+
+	constructor(page: Page) {
+		this.page = page;
+	}
+
+	get btnUploadPrivateApp(): Locator {
+		return this.page.locator('[data-qa-id="upload-private-app"]');
+	}
+
+	get btnInstallPrivateApp(): Locator {
+		return this.page.locator('[data-qa-id="install-private-app"]');
+	}
```

Comment:
Instead of `data-qa-id` could it be possible to use a combination of role and name here?